model-optimizer/setup.py

```diff
@@ -74,33 +82,42 @@ def find_package_modules(self, package, package_dir):
 packages = find_packages()
 packages = [PACKAGE_NAME + '.' + p for p in packages]
 
-setup(name='openvino-mo',
-      version='0.0.0',
-      author='Intel Corporation',
-      author_email='openvino_pushbot@intel.com',
-      url='https://github.com/openvinotoolkit/openvino',
-      packages=packages,
-      package_dir={PACKAGE_NAME: '.'},
-      py_modules=py_modules,
-      cmdclass={
-          'install': InstallCmd,
-          'build_py': BuildCmd,
-      },
-      entry_points={
-          'console_scripts': [
-              'mo = mo.__main__:main',
-           ],
-      },
-      package_data={
-        'mo.mo.front.caffe.proto': ['*.proto'],
-        'mo.extensions.front.mxnet': ['*.json'],
-        'mo.extensions.front.onnx': ['*.json'],
-        'mo.extensions.front.tf': ['*.json'],
-      },
-      classifiers=[
-        "Programming Language :: Python :: 3",
-        "License :: OSI Approved :: Apache Software License",
-        "Operating System :: OS Independent",
-      ],
-      install_requires=deps,
+setup(
+    name='openvino-mo',
+    version='0.0.0',
+    author='Intel Corporation',
+    author_email='openvino_pushbot@intel.com',
+    url='https://github.com/openvinotoolkit/openvino',
+    packages=packages,
+    package_dir={PACKAGE_NAME: '.'},
+    py_modules=py_modules,
+    cmdclass={
+        'install': InstallCmd,
+        'build_py': BuildCmd,
+    },
+    entry_points={
+        'console_scripts': [
+            'mo = mo.__main__:main',
+        ],
+    },
+    package_data={
+      'mo.mo.front.caffe.proto': ['*.proto'],
+      'mo.extensions.front.mxnet': ['*.json'],
+      'mo.extensions.front.onnx': ['*.json'],
+      'mo.extensions.front.tf': ['*.json'],
+    },
+    extras_require={
+      'caffe': read_text('requirements_caffe.txt'),
+      'kaldi': read_text('requirements_kaldi.txt'),
+      'mxnet': read_text('requirements_mxnet.txt'),
+      'onnx': read_text('requirements_onnx.txt'),
+      'tensorflow': read_text('requirements_tf.txt'),
+      'tensorflow2': read_text('requirements_tf2.txt'),
```

Comment:
Maybe just `tf` and `tf2`?

Reply:
Or in addition them,
```python
      'tensorflow': read_text('requirements_tf.txt'),
      'tensorflow2': read_text('requirements_tf2.txt'),
      'tf': read_text('requirements_tf.txt'),
      'tf2': read_text('requirements_tf2.txt'),
```

Reply:
these tags name will be reused in openvino-dev packages where should have consistent naming along with other components. OMZ tools already provides names for extras: 
https://github.com/openvinotoolkit/open_model_zoo/blob/71376c6ec0e485eb8c7c117e8754120d31369f67/tools/downloader/setup.py#L99